src/gesture-recognizers.js

```diff
@@ -742,14 +756,17 @@ export class PinchRecognizer extends GestureRecognizer {
     // for second touch.
     if (touches && touches.length == 1) {
       return true;
-    } else if (touches && touches.length == 2) {
+    }
+    if (touches && touches.length >= 2) {
```

Comment:
No need to change, but this feels a bit weird. Seems like if we don't have one touch, then either we have zero or two+. I'm not sure if you can have zero touches in a touchmove. I think this may flow better if it were:

```javascript
if (!touches || touches.length == 0) {
  return false;
}

if (touches.length == 1) {
  return true;
}

// Have 2+ touches
```

Reply:
Refactored this function for clarity. PTAL.